text/0000-crypto.md

```diff
@@ -41,6 +41,17 @@ This package will use the OpenSSL crypto library through the C FFI. The followin
     	ConstantTimeCompare(s1, s2) // returns false
    
    see also: [Golang implementation](https://golang.org/src/crypto/subtle/constant_time.go?s=490:531#L2)
+   
+- Digest
+
+    class that produces a hash from chunks of input. The hash can be calculated at any time and updated with the update method.
+    
+    Example use:
+
+        let d = Digest.sha1()
```

Comment:
Seems like this could take a primitive as a type argument to a `create` constructor, as I think @sylvanc was suggesting.  This would clear up my concern about having a big `match` statement in the implementation of `Digest`, since it could delegate to methods of the primitive.

This would look like:
```pony
let d = Digest[SHA1]
// ...
```